tests/integration_tests/ignore_update_only_columns/run.sh

```
@@ -0,0 +1,71 @@
+	run_kafka_consumer $WORK_DIR "$CONSUMER_URI" "$CUR/conf/changefeed.toml" "" ""
+
+	run_sql_file $CUR/data/data.sql ${UP_TIDB_HOST} ${UP_TIDB_PORT}
+
+	check_table_exists "ignore_update_only_columns.finish_mark" ${DOWN_TIDB_HOST} ${DOWN_TIDB_PORT} 200
+	check_result
+
+	cleanup_process $CDC_BINARY
+}
+
+trap 'stop_test $WORK_DIR' EXIT
+run $*
```

Shellcheck (0.11.0)

[warning] Use "$@" (with quotes) to prevent whitespace problems.

(SC2048)